docs/zh/connector-v2/sink/Paimon.md

```diff
@@ -6,6 +6,15 @@
 
 Apache Paimon数据连接器。支持cdc写以及自动建表。
 
+### Seatunnmel与Paimon版本对照
+
+| Seatunnel Version               | Paimon Version   |
+|---------------------------------|------------------|
+| 2.3.2.RELEASE  -  2.3.3.RELEASE | 0.4-SNAPSHOT     |
+| 2.3.4.RELEASE                   | 0.6-SNAPSHOT     |
+| 2.3.5.RELEASE  -  2.3.8.RELEASE | 0.7.0-incubating |
+| 2.3.9-SNAPSHOT                  | 0.8.2            |
```

Comment:
```suggestion
| Seatunnel Version               | Paimon Version   |
|---------------------------------|------------------|
| 2.3.2 -  2.3.3| 0.4-SNAPSHOT     |
| 2.3.4                  | 0.6-SNAPSHOT     |
| 2.3.5 -  2.3.8| 0.7.0-incubating |
| 2.3.9                 | 0.8.2            |
```
remove `RELEASE/SNAPSHOT` so that we no need to change it after release version.

Reply:
get